.salt-lint

```diff
@@ -0,0 +1,4 @@
+---
+skip_list:
```

Comment:
Good idea!

Please add a comment here referencing the info from the [official formula template](https://github.com/saltstack-formulas/salt-formula/blob/master/.salt-lint)

Reply:
added some comments to the `.salt-lint` file.